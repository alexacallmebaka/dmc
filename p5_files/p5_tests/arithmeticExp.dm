a:int;

main : () void {
  a = true + 1;
  a = "EECS" + 665;
  a = 1 + 2;

  a = true - 1;
  a = "EECS" - 665;
  a = 1 - 2;

  a = true * 1;
  a = "EECS" * 665;
  a = 1 * 2;

  a = true / 1;
  a = "EECS" / 665;
  a = 1 / 2;
}