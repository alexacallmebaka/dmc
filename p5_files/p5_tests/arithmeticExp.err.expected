FATAL [4,7]-[4,11]: Arithmetic operator applied to invalid operand
FATAL [5,7]-[5,13]: Arithmetic operator applied to invalid operand
FATAL [8,7]-[8,11]: Arithmetic operator applied to invalid operand
FATAL [9,7]-[9,13]: Arithmetic operator applied to invalid operand
FATAL [12,7]-[12,11]: Arithmetic operator applied to invalid operand
FATAL [13,7]-[13,13]: Arithmetic operator applied to invalid operand
FATAL [16,7]-[16,11]: Arithmetic operator applied to invalid operand
FATAL [17,7]-[17,13]: Arithmetic operator applied to invalid operand
Type Analysis Failed